### PR TITLE
update the changelog for ROCm6.1 release

### PR DESCRIPTION
**Pull request by illsilin**
Updating changelog for 6.1 release.

### CODE REVIEW

**Review by junliume — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,20 +2,27 @@
 
 Full documentation for Composable Kernel is not yet available.
 
-## (Unreleased) CK
+## CK for ROCm 6.1.0
```
Comment (Rmalavally) [inline CHANGELOG.md:5]: Looks good. Recommended order for consistency:

- Additions
- Optimizations
- Fixes

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,20 +2,27 @@
...
 
 ### Fixes
-None
+* Reduced the build time for most GPU architectures (#1084)
+* Fixed some conversion issues for fp8 data type (#1099)
 
 ### Optimizations
-None
+* New performance optimizations for gemm operations on MI200 and MI300 architectures (#1135)
```
Comment (Rmalavally) [inline CHANGELOG.md]: GEMM operations

Comment (illsilin) [reply CHANGELOG.md]: OK, will capitalize "GEMM".

File: CHANGELOG.md (hunk 1 above)
Comment (illsilin) [reply CHANGELOG.md:5]: For all previous releases the items came in the following order: Fixes, Optimizations, Additions. Changes. So I'm not sure what order consistency you have in mind. 
Do you recommend reshuffling these items just for this release or for all previous releases as well?

Comment (Rmalavally) [reply CHANGELOG.md:5]: Let's start with this release; for now, I have updated the ROCm 6.1 changelog manually in the correct order. 

https://github.com/Rmalavally/ROCm-internal/blob/ROCm-6.1_Release-Notes/CHANGELOG.md

Comment (illsilin) [reply CHANGELOG.md:5]: OK, sounds good.

**Review by Rmalavally — APPROVED** [review]
Looks good. Thanks, @illsilin

**Pull request merged**